src/pharmpy/tools/resmod/tool.py

```diff
@@ -121,6 +125,43 @@ def _create_power_model(input_model):
     return model
 
 
+def _create_combined_error_model(input_model):
+    combined_error_model = pharmpy.model.Model()
```

Comment:
Does this need to be created from scratch? Couldn't it use the base model as starting point?

Reply:
Sorry, I never think about this point...I can have a try!